test/unit/server/handlers/consents.test.ts

```diff
@@ -68,7 +68,7 @@ describe('server/handlers/consents', (): void => {
       expect(setImmediate).toHaveBeenCalled()
     })
 
-  it('Should return 400 code due to invalid request',
+  xit('Should return 400 code due to invalid request',
```

Comment:
Is this a typo?

Reply:
I will add comment that test is not required now, as we removed participant ID from body.
That validates whether request is valid by comparing if source header matches participant ID